### ISSUE TITLE
Loading non-HTTPS assets into HTTPS pages

### ISSUE THREAD

**Issue by firnsy** [makehackvoid/makehackvoid.com#118]
The openlayers script on the MHV site is being loaded from a non-https location (because the upstream  doesn't appear to provide one) and giving mixed content errors in chrome.

```
<script src="http://www.openlayers.org/api/OpenLayers.js"></script>
```

Self hosting the asset is probably the quickest solution.

**Comment by jamiereid** [comment]
Solved by PR #121

**Comment by jamiereid** [comment]
Commit 9bdf4b3 solves this.
